introspection-engine/connectors/mongodb-introspection-connector/tests/index/mod.rs

```diff
@@ -592,7 +592,7 @@ fn multi_column_fulltext_index_with_asc_in_end() {
           name  String
           title String
 
-          @@fulltext([name, title, age(sort: Asc)], map: "name_\"text\"_title_\"text\"_age_1")
```

Comment:
I see this as a bug fix, but to be confirmed.

Reply:
We should really reconsider how this works on mongo... The thing is you can create an index that has parts as a B-Tree, then some field or fields are a fulltext index. This example here has the fields `name` and `title` as a fulltext index, and `age` as B-Tree. Now if you do not have the `sort` defined, the index is different and considered as a fulltext index combining all three fields.

In Mongo terms this index here would be:

```json
{ "name": "text", "title": "text", "age": 1 }
```

Which is a valid index definition. Now, if we remove the `sort` param, it would be:

```json
{ "name": "text", "title": "text", "age": "text" }
```